master/security/adopt-zero-trust.md

```diff
@@ -0,0 +1,314 @@
+
+At a high level, you will undertake the following steps to establish a Zero Trust Network:
+
+1. Install Calico.
+1. Install Istio and enable Calico integration.
+1. Establish workload identity by using Service Accounts.
+1. Write initial whitelist policies for each service.
+
+After your Zero Trust Network is established, you will need to maintain it. 
+
+#### Install Calico
+
+Follow the [install instructions]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/) to get Calico software running in your cluster.
+
+#### Install Istio and enable Calico integration
+
+Follow the instructions to enable Calico Application Layer Policy.
+
+The instructions include a “demo” install of Istio for quickly testing out functionality. For a production installation to support a Zero Trust Network, you should instead follow the official Istio install instructions. Be sure to enable mutually authenticated TLS (mTLS) in your install options by setting **global.mtls.enabled to true**.
+
+#### Establish workload identity by using Service Accounts
+
+Our eventual goal is to write access control policy that authorizes individual expected network flows. We want these flows to be scoped as tightly as practical.  In a Calico Zero Trust Network, the cryptographic identities are Kubernetes Service Accounts. Istio handles crypto-key management for you so that each workload can assert its Service Account identity in a secure manner.
+
+You have some flexibility in how you assign identities for the purpose of your Zero Trust Network policy. The right balance for most people is to use a unique identity for each Kubernetes Service in your application (or Deployment if you have workloads that don’t accept any incoming connections). Assigning identity to entire applications or namespaces is probably too granular, since applications usually consist of multiple services (or dozens of microservices) with different actual access needs.
+
+You should assign unique identities to microservices even if you happen to know that they access the same things. Your policy will be more readable if the identities correspond to logical components of the application. You can grant them the same permissions easily, and if in the future they need different permissions it will be easier to handle.
+
+After you decide on the set of identities you require, create the Kubernetes Service Accounts, then modify your application configuration so that each Deployment, ReplicaSet, StatefulSet, etc. uses the correct Service Account.
+
+#### Write initial whitelist policies for each service
+
+The final step to establishing your Zero Trust Network is to write the policies for each service in your network. The Application Layer Policy Tutorial gives an overview of setting up policies that allow traffic based on Service Account identity.
```

Comment:
Application Layer Policy Tutorial should be a link to that page in the docs

Reply:
Added.